docs/actions/transformer_actions/README.md

```diff
@@ -0,0 +1,772 @@
+2. Create a transformer using the transformer action that corresponds to your template
+3. Fill in the parameters for the transformer action to build your transformer action payload
+4. Build a `BaseAction` object using the payload and execute the action to get the transformed data frame
+
+
+## Core Concepts
+
+The core object of any transformer action is the **payload**, a JSON object that describes the transformation to perform alongside settings for that transformation. A sample transformer action payload is shown below:
+
+```json
+{
+    "action_type": "impute",
+    "action_arguments": ["age", "name"],
+    "action_options": {
+        "strategy": "constant",
+        "value": "32"
+    },
+    "axis": "column",
+    "action_variables": {
+        "age": {
+            "uuid": "age",
+            "column_type": "number"
+        },
+        "name": {
+            "uuid": "name",
+            "column_type": "text"
+        }
+    }
+}
+```
+The above transformer action imputes missing values in the 'age' and "name" columns with the constant value `32`. Here is a list of the keys used in transformer action payloads:
+
+| Key              | Description                                                                                                                                                                         |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| axis             | Axis to perform the transformation on. Either 'row' or 'column'.                                                                                                                    |
+| action_type      | Type of action to perform. See [Transformer Action Reference](#transformer-action-reference) for the list of action types.                                                          |
+| action_code      | Text field for specifying any code such as filters.                                                                                                                                 |
+| action_arguments | The columns to perform the transformation on.                                                                                                                                       |
+| action_options   | Other settings to provide to the transformer action. See [Transformer Action Reference](#transformer-action-reference) for specific details on which actions require which options. |
+| action_variables | Variable metadata for each input argument (column). Specifies column name and type.                                                                                                 |
+| outputs          | Variable metadata for each output column. Specifies column name and type.                                                                                                           |
+
```

Comment:
Let's also explain what column_type is.

Reply:
Added an appendix with table of all possible column types and an explanation.